src/System Application/App/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -43,6 +43,16 @@ codeunit 9999 "Upgrade Tag"
         exit(UpgradeTagImpl.HasUpgradeTag(Tag, TagCompanyName));
     end;
 
+    /// <summary>
+    /// Verifies if the database upgrade tag exists.
+    /// </summary>
+    /// <param name="Tag">Tag code to check</param>
+    /// <returns>True if the Tag with given code exist.</returns>
```

Comment:
I know that you only copied the existing pattern. However, I see no reason to copy grammatical errors as well:

```suggestion
    /// <returns>True if the Tag with the given code exists.</returns>
```